Add original keymap made by FKML

FKML520 opened the pull request:
I add my keymap based on “swammatch/qmkfirmware” and send to “add silverbullet44”.

Commit:
Add original keymap made by FKML

swanmatch commented:
キーマップありがとー！！！
いま出先で画面ちっちゃくてキーマップよく見えないけど、とりあえず見れる範囲でみましゅ！

swanmatch commented on keyboards/silverbullet44/keymaps/FKML/config.h:
```diff
@@ -0,0 +1,19 @@
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
```
このファイルいらないらしい…

swanmatch commented on keyboards/silverbullet44/keymaps/FKML/keymap.c:
```diff
@@ -0,0 +1,169 @@
+/* Copyright 2019 SwanMatch
```
せっかくだから、ふかめるさんのCopyrightにしてみては?
```suggestion
/* Copyright 2020 FKML
```

swanmatch commented on keyboards/silverbullet44/keymaps/FKML/keymap.c:
```diff
@@ -0,0 +1,169 @@
+//  QMKBEST = SAFE_RANGE,
+//  QMKURL
+//};
+
```
消しちゃえー！

swanmatch commented on keyboards/silverbullet44/keymaps/FKML/keymap.c:
```diff
@@ -0,0 +1,169 @@
+#endif
+
+int RGB_current_mode;
+bool alt_pressed = false;
```
消しちゃえー！
```suggestion
```

swanmatch commented on keyboards/silverbullet44/keymaps/FKML/keymap.c:
```diff
@@ -0,0 +1,169 @@
+          layer_off(_ADJUST);
+        }
+        return false;
+        break;
```
ちょっとちゃんと見れてないんだけど、update_tri_layerとADJUSTのコード両方つかってる？
どっちかでよかったらどっちかにしとくと良いかも。

swanmatch commented on keyboards/silverbullet44/keymaps/FKML/keymap.c:
```diff
@@ -0,0 +1,169 @@
+ *                            | CAPS  | LOWER| Space/  Bcsp /     \    .  \ Ent | RAISE|LEDOff|
+ *                            `--------------- -------------'       '--------------------------'
+ */
+ [_QWERTY] = LAYOUT( \
```
なんか全体的に行末のバックスラッシュいらないって言われた。。。
(魔王知らなかった)

```suggestion
 [_QWERTY] = LAYOUT(
```

swanmatch commented on keyboards/silverbullet44/keymaps/FKML/keymap.c:
```diff
@@ -0,0 +1,169 @@
+
+void led_set_user(uint8_t usb_led) {
+
+}
```
消しちゃえー！

swanmatch reviewed (changes requested):
拝啓 創造神さまへ

おめでとう！
このレポジトリの記念すべき最初のプルリクです！！

実は創造神さまがcloneしたあとに本家にプルリク送ったらけちょんけちょんにされました。
そのへんの指摘受けて直したのがadd_silverbullet44ってブランチにいます。

魔王がけちょんけちょんにされている現場はこちらになります。
https://github.com/qmk/qmk_firmware/pull/7950

いずれ本家にマージする際に同様の指摘を受けると思うのだよねー
もともと私のキーマップの不備なので、
めんどかったらこのままマージしたあとおれのほうで直して本家にがっちゃんこするけど、
なんか人のキーマップ勝手にいじってるみたいになっちゃうから、
ここで直しちゃったほうが良い気がするー

masterとadd_silverbullet44の違いは下記コマンドでわかるよん。
git diff master..add_silverbullet44 keyboards/silverbullet44/keymaps/default/

それではよいキーボードライフを！

Commit:
deleated at config.h

Commit:
Changed Copyright

Commit:
Delete
// Defines the keycodes used by our macros in process_record_user
//enum custom_keycodes {
//  QMKBEST = SAFE_RANGE,
//  QMKURL
//};

Commit:
Delete bool alt_pressed = false;

Commit:
Delete ADJUST

Commit:
Delete /

Commit:
Delete void matrix_scan_user(void) {

}

void led_set_user(uint8_t usb_led) {

}

FKML520 commented:
拝啓　魔王様

ご丁寧なご指摘ありがとうございます。
当方キィボォド初心者につき大変勉強になりました。

ご指摘いただいた件に関しては全て修正し、ビルドができることも確認致しました。

けちょんけちょんにされても負けるなーがんばれー！

創造神

Commit:
Change Copyright's year.

swanmatch commented on keyboards/silverbullet44/keymaps/FKML/keymap.c:
```diff
@@ -0,0 +1,148 @@
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
```
ADJUSTを使わないようなら消してくれると…！
```suggestion
```

swanmatch commented on keyboards/silverbullet44/keymaps/FKML/keymap.c:
```diff
@@ -0,0 +1,148 @@
+  KC_GRV , KC_EXLM,   KC_AT, KC_HASH,  KC_DLR, KC_PERC,                  KC_CIRC,  KC_AMPR,  KC_ASTR,  KC_LPRN, KC_RPRN, KC_PIPE, 
+  _______, _______, _______, _______, _______, _______,                     KC_1,     KC_2,     KC_3,     KC_4,    KC_5, _______, 
+  _______, _______, _______, _______, _______, _______,                     KC_6,     KC_7,     KC_8,     KC_9,    KC_0, _______, 
+                             _______, _______, _______, ADJUST,  _______, _______, _______, _______ 
```
ADJUSTを使わないようなら消してくれると…！
```suggestion
                             _______, _______, _______, _______,  _______, _______, _______, _______ 
```

swanmatch commented on keyboards/silverbullet44/keymaps/FKML/keymap.c:
```diff
@@ -0,0 +1,148 @@
+  _______,   KC_F1,   KC_F2,    KC_F3,   KC_F4,  KC_F5,                     _______, _______, _______, _______,  KC_MINS, KC_PLUS, 
+  _______,   KC_F6,   KC_F7,    KC_F8,   KC_F9,  KC_F10,                    _______, _______, _______, _______,  KC_LBRC, KC_RBRC, 
+  _______,  KC_F11,   KC_F12, _______, _______, _______,                    _______, _______, KC_LEFT,   KC_UP,  KC_DOWN, KC_RGHT, 
+                              _______, _______, _______, _______,   ADJUST, _______, _______, _______ 
```
同上～
```suggestion
                              _______, _______, _______, _______,   _______, _______, _______, _______ 
```

swanmatch commented on keyboards/silverbullet44/keymaps/FKML/keymap.c:
```diff
@@ -0,0 +1,148 @@
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 3
+
```
ごめん、これはこの前の指摘漏れ！
列挙型って言うんだけ？にしとくといいってさ。

```suggestion
enum layer {
    _QWERTY,
    _CURSOL,
    _CALC,
    _ADJUST,
};
```

swanmatch commented on keyboards/silverbullet44/keymaps/FKML/readme.md:
```diff
@@ -0,0 +1,51 @@
+# The default keymap for silverbullet44
+
+
```
全体を「```」で囲って、行頭の*とかを消すと見やすくなってオススメ。
参考↓
https://raw.githubusercontent.com/swanmatch/qmk_firmware/add_silverbullet44/keyboards/silverbullet44/keymaps/default/readme.md
https://github.com/swanmatch/qmk_firmware/tree/add_silverbullet44/keyboards/silverbullet44/keymaps/default

swanmatch commented on keyboards/silverbullet44/keymaps/FKML/keymap.c:
```diff
@@ -0,0 +1,148 @@
+  ADJUST,
+};
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
```
これももうデフォで定義されてるからいらないらしいよ…
便利な世の中になったものだ。
```suggestion
```

swanmatch reviewed (commented):
さっそく修正ありがとう！
追加でいくつか指摘させていただきましたので、
お手すきの際にご確認いただけますでしょうか。
これ終わったらマージしちゃうべ！

追記:
おとといの帰りの電車、先頭車両に乗ったけど、
カーテン閉めていてあまりおもしろくなかったです。
残念。。。

swanmatch commented on keyboards/silverbullet44/keymaps/FKML/keymap.c:
```diff
@@ -0,0 +1,148 @@
+        }
+        return false;
+        break;
+        case RAISE:
```
インデント
あと、ここだけインデントが2spaceになってるので
4spaceにしとくと良さそう。

```suggestion
      case RAISE:
```

swanmatch reviewed (changes requested):
ごめん、もういっこ！

Commit:
Delete adjust

Commit:
Delete adjust

Commit:
Delete adjust

Commit:
列挙型に変更

Commit:
Enclose  keymap with ```.

Commit:
Delete
#define _______ KC_TRNS
#define XXXXXXX KC_NO

Commit:
Fix indentation.

Commit:
Delete some rows.

FKML520 commented:
魔王様

追加でいただいてた修正分お待たせいたしました！
修正できましたー。確認お願いいたしますー！！！

カーテン閉まってたのは残念だ…　反射を防ぐために閉めちゃってたのかしらん。

Commit:
Aligned columns of ''readme.md''.

swanmatch commented:
かみさま

修正ありがとう！マージしました！
本家マージの前にインデントと行末空白とかだけ
こっちで若干整形するかも～

今後ともSilverBullet44をかわいがってあげてください。

swanmatch commented:
ちなみに整形↓の感じです～
https://github.com/swanmatch/qmk_firmware/commit/acfc83d0ebbf5c307c7c1d262a0971653b212bc6#diff-d4b018956f579d6120d071a277b126c6R66

FKML520 commented:
魔王さま

やった！まーじ！人(神)生初のまーじです！
整形ありがとうございます。お手間おかけしてすみません。。。

SilverBullet44、めちゃめちゃ可愛がります…！！！！！！ぴこぴこぴこ！
